features/steps/partcad-cli/commands/list/packages.py

```
@@ -0,0 +1,22 @@
+import shutil
+
+
+@given('I have temporary $HOME in "{directory}"')
+def step_impl(context: Context, directory):
+    # TODO: @alexanderilyin: use dedicated object for test directories
+    os.makedirs(directory, exist_ok=True)
+    context.home_dir = tempfile.mkdtemp(
+        prefix="partcad-cli-",
+        dir=directory,
+    )
+    logging.info(f"Created temporary home: {context.home_dir}")
```

Ruff (0.8.0)

Logging statement uses f-string

(G004)